Lombiq.HelpfulLibraries.Common/Extensions/EnumerableExtensions.cs

```diff
@@ -6,6 +6,29 @@ namespace System.Collections.Generic;
 
 public static class EnumerableExtensions
 {
+    /// <summary>
+    /// Same as <see cref="Enumerable.Aggregate{TSource}"/>, but keeps the same <paramref name="seed"/> so the <paramref
+    /// name="action"/> doesn't have to return anything.
+    /// </summary>
+    /// <param name="source">The collection to traverse.</param>
+    /// <param name="seed">The object to pass to <paramref name="action"/> during enumeration.</param>
+    /// <param name="action">The action called on each item of <paramref name="source"/>.</param>
+    /// <typeparam name="TSource">The type of the items in <paramref name="source"/>.</typeparam>
+    /// <typeparam name="TAccumulate">The type of the <paramref name="seed"/>.</typeparam>
+    /// <returns>The instance passed to <paramref name="seed"/>.</returns>
+    public static TAccumulate AggregateSeed<TSource, TAccumulate>(
```

Comment:
Is it still an aggregate now? Looks more like chaining calls on the same target object. Maybe rename to `Chain`?

Reply:
It's still aggregate, it just aggregates the data into the seed object. In my experience this is 99.999% of the uses cases for Aggregate anyway. The only time you need to return anew value is if the seed is immutable or a value type, that's the reason I added the `class` type constraint. So this is just a shortcut to make things neater.

It's bad enough that C# calls it "aggregate" instead of "reduce" like most other languages (e.g. JS, C++, Python, Rust, PHP, etc) which I think is a big part why it's so rarely used. Naming this method yet something else would be really bad for discoverability.

Reply:
I see, thanks for the clarification.

I agree on the terrible naming. It's probably the last LINQ method I started using, and I still use it very rarely.